<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-16</x>
     <y>-17</y>
     <width>842</width>
     <height>686</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 251, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>16</y>
      <width>825</width>
      <height>266</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 52, 52);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>21</y>
       <width>298</width>
       <height>67</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#c8c8c8;&quot;&gt;Payment Page&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="movieName">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>98</y>
       <width>351</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 12pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#c8c8c8;&quot;&gt;Movie:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="timing">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>151</y>
       <width>411</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 12pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#c8c8c8;&quot;&gt;Timings:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="seatsLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>199</y>
       <width>441</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 12pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#c8c8c8;&quot;&gt;Seats:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="costLabel">
     <property name="geometry">
      <rect>
       <x>474</x>
       <y>143</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 12pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#c8c8c8;&quot;&gt;You need to pay:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="subtotal">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>179</y>
       <width>298</width>
       <height>67</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	padding: 10;
	color: rgb(0, 85, 127);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#c8c8c8;&quot;/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>-7</x>
      <y>532</y>
      <width>831</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 52, 52);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="proceedButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>29</y>
       <width>230</width>
       <height>38</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	border-radius:15px;
	background-color: rgb(145, 177, 194,150);
	font: 14pt &quot;Nasalization Rg&quot;;
}
QPushButton:hover{
	background-color: rgb(145, 177, 194,100);
}
QPushButton:pressed{
	background-color: rgb(145, 177, 194,80);
}</string>
     </property>
     <property name="text">
      <string>Proceed &gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>31</y>
       <width>230</width>
       <height>38</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
	border-radius:15px;
	background-color: rgb(145, 177, 194,150);
	font: 14pt &quot;Nasalization Rg&quot;;
}
QPushButton:hover{
	background-color: rgb(145, 177, 194,100);
}
QPushButton:pressed{
	background-color: rgb(145, 177, 194,80);
}</string>
     </property>
     <property name="text">
      <string>&lt; Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="YearRating_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>304</y>
      <width>172</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	padding: 10;
	color: rgb(52, 52, 52);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#343434;&quot;&gt;Card Number:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>344</y>
      <width>366</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
 	border : 2px solid rgb(207, 210, 213);
	padding:10px;
	
	font: 11pt &quot;Nasalization Rg&quot;;
}</string>
    </property>
    <property name="inputMask">
     <string>9999{    -    }9999{    -    }9999{    -    }9999</string>
    </property>
    <property name="maxLength">
     <number>43</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="YearRating_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>305</y>
      <width>172</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	padding: 10;
	color: rgb(52, 52, 52);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#343434;&quot;&gt;CVV Number:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>343</y>
      <width>167</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
 	border : 2px solid rgb(207, 210, 213);
	padding:10px;
	
	font: 11pt &quot;Nasalization Rg&quot;;
}</string>
    </property>
    <property name="inputMask">
     <string>9 9 9</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="YearRating_7">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>411</y>
      <width>172</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	padding: 10;
	color: rgb(52, 52, 52);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#343434;&quot;&gt;Expiry Date:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>447</y>
      <width>126</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
 	border : 2px solid rgb(207, 210, 213);
	padding:10px;
	
	font: 11pt &quot;Nasalization Rg&quot;;
}</string>
    </property>
    <property name="inputMask">
     <string>9 9</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>447</y>
      <width>126</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
 	border : 2px solid rgb(207, 210, 213);
	padding:10px;
	
	font: 11pt &quot;Nasalization Rg&quot;;
}</string>
    </property>
    <property name="inputMask">
     <string>9 9</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="YearRating_8">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>431</y>
      <width>43</width>
      <height>82</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nasalization Rg</family>
      <pointsize>38</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	padding: 10;
	color: rgb(52, 52, 52);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#343434;&quot;&gt;\&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="YearRating_9">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>411</y>
      <width>172</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	padding: 10;
	color: rgb(52, 52, 52);
	font: 38pt &quot;Nasalization Rg&quot;;
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#343434;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>443</y>
      <width>257</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
 	border : 2px solid rgb(207, 210, 213);
	padding:10px;
	
	font: 11pt &quot;Nasalization Rg&quot;;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Pictures/res.qrc"/>
 </resources>
 <connections/>
</ui>
